core/modules/color/color.module

```diff
@@ -213,17 +213,19 @@ function color_css_alter(&$css) {
 
     if (!empty($color_paths)) {
       $info = color_get_info($theme_key);
-      foreach($color_paths as $color_path) {
 
-        foreach ($info['css'] as $file) {
-          $old_path = _color_get_file_path($file, $theme_key);
-          // Color module currently requires unique file names to be used,
-          // which allows us to compare different file paths.
-          if (backdrop_basename($old_path) == backdrop_basename($color_path)) {
-            // Replace the path to the new css file.
-            // This keeps the order of the stylesheets intact.
```

Comment:
Perhaps keep some of the comments where relevant.

Reply:
I added back some of the comments.